docs/tutorials/web.md

```diff
@@ -5,10 +5,405 @@ has_children: false
+
+- [Use AutoML to create a custom model](https://docs.microsoft.com/azure/machine-learning/concept-automated-ml)
+- [Use Custom Vision Cognitive Services to create a custom model](https://docs.microsoft.com/azure/cognitive-services/custom-vision-service/overview)
+- [Use Azure Machine Learning Designer to create a custom model](https://docs.microsoft.com/en-us/azure/machine-learning/concept-designer)
+- [Build your own model with PyTorch.](https://docs.microsoft.com/learn/paths/pytorch-fundamentals/)
+
+# Create a Static Site with NextJS (a ReactJS Framework) to deploy models in the browser
+
+## The Template
+
+The goal of this template is to provide code examples and a starting point for deploying your models. Let’s dive into the template and breakdown the code while looking at how to deploy a computer vision model. This template uses NextJS, which is an implementation of ReactJS which much like this template gives you a starting point for building production React apps. Read more about [NextJS Here](https://nextjs.org/). This template is configured with webpack, onnxruntime-web, React, TypeScript and dev environments for testing and debugging.
+
+## The `utils` Folder 
+
+There are three files in the Utils folder `imageHelper.ts`, `modelHelper.ts` and `predict.ts`. Predict is the entry point from the web component to start inferencing. Here we import the helpers and call the default functions to get the image tensor and to run our model inference. 
+
+### predict.ts
+
+```javascript
+// Language: typescript
+// Path: react-next\utils\predict.ts
+import getImageTensorFromPath from './imageHelper';
+import {runSqueezenetModel} from './modelHelper';
+
+export async function inferenceSqueezenet(path: string): Promise<[any,number]> {
+  // 1. Convert image to tensor
+  const imageTensor = await getImageTensorFromPath(path);
+  // 2. Run model
+  const [predictions, inferenceTime] = await runSqueezenetModel(imageTensor);
+  // 3. Return predictions and the amount of time it took to inference.
+  return [predictions, inferenceTime];
+}
+```
+### imageHelper.ts
+
+First, we need to get our image from path (can be local or url) and convert it to a tensor. The `getImageTensorFromPath` function in the `imageHelper.ts` uses `JIMP` to read the file, resize and return the `imageData`. [JIMP](https://www.npmjs.com/package/jimp) is a JavaScript image manipulation library. It has many built in functions for working with image data such as resizing, grey scale, write, and more. In this example we only need to resize however in your code you may need additional image data processing. 
+
+```javascript
+import Jimp  from 'jimp/*';
+import { Tensor } from 'onnxruntime-web';
+const jimp = require('jimp');
```

Comment:
[example code] should not use both "import" and "require" in one script. ( did you try `import * as Jimp from 'jimp';` ?)

Reply:
removed dup require import